docs/howto/build-run.md

```diff
@@ -204,7 +204,7 @@ might need to restart Metro / `react-native start` after doing so.
 When trying to build the Android app, you may see this error:
 
 ```
-A problem occurred evaluating project ':@remobile/react-native-toast'.
+A problem occurred evaluating project ':<project name>'.
```

Comment:
Ah, well spotted :)

I think it is slightly more helpful if this has the shape of a real example (even though which project it is doesn't really matter.) Saves any need to worry, as the reader, about whether things like `<` and `>` are part of the real error message or are meta-syntax.

For fun, here's a prediction of which project name would appear here instead:

I think the reason this error message came up this way is just that this was the first of our Gradle subprojects to get evaluated -- presumably because, at the time, it was the lexicographically-first of them, because it was the only one with a name starting with `@`. This entry was added in 4555fb22b, and:
```
$ git show 4555fb22b:android/settings.gradle | grep ^include | LC_ALL=C sort
include ':@remobile/react-native-toast'
include ':app'
include ':react-native-device-info'
include ':react-native-fetch-blob'
[...]
```

(The `LC_ALL=C` environment-variable setting causes `sort` to sort by byte values, rather than [the human-text-oriented sorting](https://en.wikipedia.org/wiki/ISO_14651) it does by default.)

So, today:
```
$ <android/settings.gradle grep ^include | LC_ALL=C sort | head -1
include ':@react-native-community_async-storage'
```
I think it would be `project ':@react-native-community_async-storage'`.

Reply:
Interesting! Thanks for that investigation.